src/utils.cpp

```diff
@@ -27,4 +31,31 @@ namespace utils {
 
         return std::string{ buffer };
     }
+
+    [[nodiscard]] std::filesystem::path get_root_folder() {
+#if defined(__ANDROID__)
+        char* pref_path = SDL_GetPrefPath("coder2k", "oopetris");
```

Comment:
We definitely need some separate file (e.g. "constants.hpp") to put those magic strings in. I will create an issue for this.
(see #79)